src/ORM/ResultSet.php

```diff
@@ -410,26 +329,6 @@ protected function _calculateColumnMap(Query $query): void
         $this->_map = $map;
     }
 
-    /**
-     * Helper function to fetch the next result from the statement or
-     * seeded results.
-     *
-     * @return mixed
-     */
-    protected function _fetchResult(): mixed
-    {
-        if ($this->_statement === null) {
-            return false;
-        }
-
-        $row = $this->_statement->fetch('assoc');
-        if ($row === false) {
-            return $row;
-        }
-
-        return $this->_groupResult($row);
-    }
```

Comment:
I like that the ResultSet is simpler now and doesn't depend on statements.

Reply:
Yup ResultSet is a lot simpler now.